src/components/post-add-form/post-add-form.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { fetchPosts } from '../actions';
+import { withBlogService } from '../hoc';
+
+import './post-add-form.css';
+
+class PostAddForm extends Component {
+  state = {
+    title: '',
+    text: ''
+  };
+
+  handleChange = ({ target }) => {
+    const currentTarget = target.name === 'postadd' ? 'text' : 'title';
```

Comment:
Instead, you can just add property `name` to `input` and `textarea`, and then use it:

```
this.setState({
      [target.name]: target.value
    });
```